community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/pipes/CreateNodePipe.scala

```diff
@@ -48,26 +50,26 @@ case class CreateNodePipe(src: Pipe, key: String, labels: Seq[LazyLabel], proper
     context += key -> node
   }
 
- private def setProperties(context: ExecutionContext, state: QueryState, nodeId: Long) = {
-   properties.foreach { expr =>
-     expr(context)(state) match {
-       case _: Node | _: Relationship => throw new
-           CypherTypeException("Parameter provided for node creation is not a Map")
-       case IsMap(f) =>
-         val propertiesMap: Map[String, Any] = f(state.query)
-         propertiesMap.foreach {
-           case (k, v) =>
-             //do not set properties for null values
-             if (v != null) {
-               val propertyKeyId = state.query.getOrCreatePropertyKeyId(k)
-               state.query.nodeOps.setProperty(nodeId, propertyKeyId, makeValueNeoSafe(v))
-             }
-         }
-       case _ =>
-         throw new CypherTypeException("Parameter provided for node creation is not a Map")
-     }
-   }
- }
+  private def setProperties(context: ExecutionContext, state: QueryState, nodeId: Long) = {
```

Comment:
I don't think I have. The diff here shows I changed from 1 space to 2 spaces, did you read it wrong perhaps?